src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  background-color: black;
+}
+
+.container {
+  width: 75%;
+  margin: auto;
+  background-color: black;
+}
+
+.tasks-wraper {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  padding: 20px;
+}
+
+.header {
+  font-weight: bolder;
+  text-align: center;
+  font-size: x-large;
+}
+
+.add-item {
+  width: 90%;
+  height: 1rem;
+  margin: auto;
+}
+
+.task {
+  list-style-type: none;
+  margin: 1px 0;
+  border: black 1px;
+  padding: 1rem;
+  border-bottom: 1px solid #eee;
+}
+
+.tasks-item {
+  align-items: center;
+  padding: 1rem;
+  border-bottom: 1px solid #eee;
+}
+
+.checkbox {
+  margin-right: 5px;
+}
```

Comment:
- [ ] For this project, there's a specific styling template stated to follow. Kindly use your created CSS to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).  Please note that the video has more features than the current project. Therefore, you don't need to style all the features not implemented yet in the video.